api/src/main/resources/org/openmrs/liquibase/updates/liquibase-update-to-latest-2.5.x.xml

```diff
@@ -268,4 +268,21 @@
 								 referencedTableName="concept" referencedColumnNames="concept_id" />
 	</changeSet>
 
+	<changeSet id="TRUNK-6045" author="pmanko">
+		<preConditions onFail="MARK_RAN">
+			<not>
+				<columnExists tableName="location" columnName="type" />
+			</not>
+		</preConditions>
+		<comment>Adding type field to the Location table</comment>
+		<addColumn tableName="location">
+			<column name="type" type="int">
+				<constraints nullable="true"/>
+			</column>
+		</addColumn>
+		<addForeignKeyConstraint constraintName="location_type_fk"
+		baseTableName="location" baseColumnNames="type"
+		referencedTableName="concept" referencedColumnNames="concept_id" />
```

Comment:
nit: it'd be nice if this was formatted like the change set above it, but that's not too much of an issue

minor: I think it would be better to have a more descriptive column name, e.g. `location_type_concept_id`